test/fixtures/console/core_line_numbers.snapshot

```diff
@@ -0,0 +1,10 @@
+node:punycode:*
+	throw new RangeError(errors[type]);
+	^
+
+RangeError: Invalid input
```

Comment:
lets avoid this PR from containing snapshots with a delta larger than the git rename threshold

Reply:
Apologies for my ignorance @MoLow but I don't know what this means. Could you please explain what you mean? Thanks!

Reply:
if the diff (either via GitHub UI, or via git diff) detects a file as new instead as renamed, I think we should exclude it from this PR, to further PRs, so reviewing would be easier

Reply:
I see. Should I make a correction and break down the PR? or is this a request for future PR's?

Thanks!

Reply:
it seems like the diff in this screenshot is a diff for a specific commit, not for the entire PR